### PR TITLE
Feature/xmldoc stratis.bitcoin.fullnode

### PR DESCRIPTION
**Pull request by Aprogiena**
XMLDOC for Stratis.Bitcoin FullNode/IFullNode

### CODE REVIEW

File: Stratis.Bitcoin/FullNode.cs [hunk 1]
```diff
@@ -1,286 +1,336 @@
...
+using System.Threading.Tasks;
 
 namespace Stratis.Bitcoin
 {
+    /// <summary>
+    /// Node providing all supported features of the blockchain and its network.
+    /// </summary>
+    public class FullNode : IFullNode
+    {
+        /// <summary>Instance logger.</summary>
```
Comment (bokobza) [inline Stratis.Bitcoin/FullNode.cs]: Logger instance would be better :-)

**Comment by bokobza** [comment]
@Aprogiena only one comment and then a little conflict. :-)

File: Stratis.Bitcoin/FullNode.cs (hunk 1 above)
Comment (Aprogiena) [inline Stratis.Bitcoin/FullNode.cs]: I know what you mean, but I use this as a technical term. Please check [Using Logging](https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/Documentation/using-logging.md) - I define there Instance logger and Class logger as two different things. This is Instance logger, so in fact this is "Instance logger's instance". 

If you are happy with the terms, the comment should be OK, if you think the comment should be fixed, possibly the terms themselves are not OK.

Comment (bokobza) [inline Stratis.Bitcoin/FullNode.cs]: ah ah ok it means the logger for this instance, not an instance of  a Logger.
What about "this instance's logger"?
If not then leave it as it is, it's fine.

Comment (Aprogiena) [inline Stratis.Bitcoin/FullNode.cs]: If you don't really mind, I would say not (it is like this elsewhere already).

The meaning is supposed to be "logger for an instance of this object" vs "logger for this class".

Comment (bokobza) [inline Stratis.Bitcoin/FullNode.cs]: yeah I understood the second part.

**Pull request merged**